### PR TITLE
Updated COLLECTIONS_4.rst to include ansible-inclusion repo link for new collection submission

### PR DESCRIPTION
**Pull request by Ompragash**
##### SUMMARY

Updated COLLECTIONS_4.rst to include ansible-inclusion repo link for new collection submission

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/roadmap/COLLECTIONS_4.rst

### CODE REVIEW

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Release schedule
 =================
 
 
-:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4.
+:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4. Request to include new collection can be submitted `here <https://github.com/ansible-collections/ansible-inclusion>`_.
```
Comment (felixfontein) [inline docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: We should avoid using non-descriptive link titles such as `here`. Maybe this is better:
```suggestion
:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4. `Requests to include new collections can be submitted <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
```
(I also adjusted the link to directly create a new discussion.)

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst [hunk 2]
```diff
@@ -12,7 +12,7 @@ Release schedule
 =================
 
 
-:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4.
+:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4. Request to include new collection can be submitted to this `github discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: GitHub Discussion

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst [hunk 3]
```diff
@@ -12,7 +12,7 @@ Release schedule
 =================
 
 
-:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4.
+:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4. Request to include new collection can be submitted to this `GitHub Discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
```
Comment (samccann) [inline docs/docsite/rst/roadmap/COLLECTIONS_4.rst]:
```suggestion
:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4. Submit a request to include a new collection in this `GitHub Discussion <https://github.com/ansible-collections/ansible-inclusion/discussions/new>`_.
```

**Review by felixfontein — APPROVED** [review]

**Review by gundalow — APPROVED** [review]
Good idea, thank you.

**Review by samccann — APPROVED** [review]

**Pull request merged**